### PR TITLE
Branding & Audience section

### LINKED ISSUE #10: [Handover] Define Project Goal & Audience
Within the project, we've performed research in order to understand the goal and the userbase of the project.

You'll need to write down this info in `audience.md` and `goal.md`. Some thing we've done:
- The persona
- The branding _(Style, color, fonts, etc...)_
- Definition of the userbase
- Goal of the project

### PR DESCRIPTION
**Pull request by BT-Creator**
This section basically describes the branding & the target of the audience for IMEC. You can also see it as a recap of what happend in week 1

### CODE REVIEW

File: project/audience.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+---
+# For whom?
+
+The LOAMA project is created for users with a technological knowledge, but we've focused on several factors while designing the application:
+- **Ease of use**: LOAMA was designed with ease of use in mind by reducing & simplifying the steps that need to be taken to assign permissions. 
+- **Accessibiliy**: In order to make LOAMA as accessible as possible, we omitted using technical terms to describe function. Instead, by using a combination of icons, simple sentences and colors, we indicate the status of files and divert attention of the user.
+
+By using the information gained during the [brand creation](/project/brand.md), we made a persona to guide us during the design process of [Loama](/loama/index.md):
+
+## Jermery
```
Comment (elroyK) [inline project/audience.md:15]: Jeremy?

File: project/audience.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+### Pains & Problems
+> What are the problems the user encounters?
+#### LOAMA
+- I can't see what permissions are configured for the files in my pod
+- I don't see who can access my files and which permissions they have
+- What do the permissions actually mean within my Solid Pod?
+- Why should I use Solid over other technologies or just having a company have my data?
+
+#### Solid-enabled apps
+- What do the permissions actually mean within my Solid Pod?
```
Comment (elroyK) [inline project/audience.md]: repeated from line 29

File: project/audience.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- How do I specify which data an application may use?
+
+### Needs
+> What does the users want to do within the app?
+#### LOAMA
+- I want to see what data an person, company or application can access
```
Comment (elroyK) [inline project/audience.md]:
```suggestion
- I want to see what data a person, company or application can access
```

File: project/audience.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- I want to see when my data was accessed
+- I want to revoke or allow acess to my data
+- I want to understand what the permissions means and the effect of when I allow them
```
Comment (elroyK) [inline project/audience.md]:
```suggestion
- I want to understand what the permissions mean and the effect of allowing them
```

File: project/audience.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+title: For whom?
+next:
+    link: /loama/index
+    text: 'Loama'
+---
+# For whom?
+
+The LOAMA project is created for users with a technological knowledge, but we've focused on several factors while designing the application:
```
Comment (Zelzahn) [inline project/audience.md]:
```suggestion
The LOAMA project is created for users with a technological background, but we've focused on several factors while designing the application:
```

File: project/audience.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+#### Solid-enabled apps
+- I want to use my data in my Solid pod across different platforms
+- I want to sign on using my Solid pod and allow the application to use my data
+
+### Motivation
+> Why does a user wants the things he wants?
+
+- I want to be in control of my own data
+- I want to share the same data across platforms
+- I want to easily disabled shared access to sensitive information
```
Comment (elroyK) [inline project/audience.md]:
```suggestion
- I want to easily revoke access to sensitive information
```

File: project/audience.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+title: For whom?
+next:
+    link: /loama/index
+    text: 'Loama'
+---
+# For whom?
+
+The LOAMA project is created for users with a technological knowledge, but we've focused on several factors while designing the application:
+- **Ease of use**: LOAMA was designed with ease of use in mind by reducing & simplifying the steps that need to be taken to assign permissions. 
+- **Accessibiliy**: In order to make LOAMA as accessible as possible, we omitted using technical terms to describe function. Instead, by using a combination of icons, simple sentences and colors, we indicate the status of files and divert attention of the user.
```
Comment (Zelzahn) [inline project/audience.md]:
```suggestion
- **Accessibiliy**: To make LOAMA as accessible as possible, we omitted using technical terms to describe functions. Instead, by using a combination of icons, simple sentences and colours, we indicate the status of files and divert the attention of the user.
```

File: project/brand.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+---
+title: Branding
+---
+# Branding
+
+For the project, we made an appropriate branding that is based on the [SOLID logo](https://solidproject.org). We did this by using several branding techniques commonly used to create a brand from scratch.
+
+## 1. Personifying the product
+In order to differentiate our brand from Solid, we trying personifying our product by asking some thought-provoking questions:
+- If our product was a person, what would they wear to a party, how would they act on the party and what would be their favorite music?
```
Comment (elroyK) [inline project/brand.md]:
```suggestion
- If our product was a person, what would they wear to a party, how would they act at the party and what would be their favorite music?
```

File: project/brand.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+- If we need to associate our product with a season, what would this be?
```
Comment (elroyK) [inline project/brand.md]:
```suggestion
- If we had to associate our product with a season, which would it be?
```

File: project/audience.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+- **Gender**: Male
+- **Education**: Bachelor in Journalism
+
+### Pains & Problems
+> What are the problems the user encounters?
+#### LOAMA
+- I can't see what permissions are configured for the files in my pod
+- I don't see who can access my files and which permissions they have
+- What do the permissions actually mean within my Solid Pod?
+- Why should I use Solid over other technologies or just having a company have my data?
```
Comment (Zelzahn) [inline project/audience.md]:
```suggestion
- Why should I use Solid over other technologies or just letting a company have my data?
```

File: project/audience.md (hunk 3 above)
Comment (Zelzahn) [inline project/audience.md]:
```suggestion
- I want to see what data a person, company or application can access
```

File: project/audience.md (hunk 4 above)
Comment (Zelzahn) [inline project/audience.md]:
```suggestion
- I want to understand what the permissions mean and the effect of when I allow them
```

File: project/brand.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+## 2. Term association
+In order to find out which things or terms we associated with our product, we created a mindmap with terms that we associate with our product and objects we associate with those terms.
```
Comment (elroyK) [inline project/brand.md]: Rephrase to avoid repeating "terms we associated with our product" :)

File: project/audience.md (hunk 6 above)
Comment (Zelzahn) [inline project/audience.md]:
```suggestion
- I want to easily disable shared access to sensitive information
```

File: project/brand.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+![LOAMA Mindmap](../images/loama-mindmap.png)
+
+This gave us more insight in what we could use as a basis for naming our product.
```
Comment (elroyK) [inline project/brand.md]:
```suggestion
This gave us more insight into what we could use as a basis for naming our product. 
```

File: project/brand.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
+---
+title: Branding
+---
+# Branding
+
+For the project, we made an appropriate branding that is based on the [SOLID logo](https://solidproject.org). We did this by using several branding techniques commonly used to create a brand from scratch.
+
+## 1. Personifying the product
+In order to differentiate our brand from Solid, we trying personifying our product by asking some thought-provoking questions:
```
Comment (Zelzahn) [inline project/brand.md]:
```suggestion
To differentiate our brand from Solid, we tried personifying our product by asking some thought-provoking questions:
```

File: project/brand.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+![LOAMA Mindmap](../images/loama-mindmap.png)
+
+This gave us more insight in what we could use as a basis for naming our product. 
+
+## 3. Name
+The name LOAMA is an acrynom:
+> **L**inked **O**pen **A**ccess **M**anagement **A**pp 
+
+It's a technical way to describe our project, but the acronym *(Loama)* was an unexpected inspiration for our brand, as it became a basis for which we build our brand, crest and color palette.
```
Comment (elroyK) [inline project/brand.md]: Avoid repetition
```suggestion
It's a technical way to describe our project, but the acronym *(Loama)* was an unexpected inspiration for our project's identity, as it became a basis for which we build our brand, crest and color palette.
```

File: project/brand.md (hunk 8 above)
Comment (Zelzahn) [inline project/brand.md]:
```suggestion
- If our product was a person, what would they wear to a party, how would they act on the party and what would their favorite music be?
```

File: project/brand.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+
+![Loama Crest](/loama-logo.svg)
+
+## 4. Typography & Color Usage
+During the design process, we research and define a typography & color palette in order to have a consistent look & feel through the project. These elements were re-used within the design prototype, presentations, live demos and documentation.
```
Comment (Zelzahn) [inline project/brand.md]:
```suggestion
During the design process, we researched and defined a typography & color palette in order to have a consistent look & feel through the project. These elements were re-used within the design prototype, presentations, live demos and documentation.
```

File: project/brand.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+
+![Color Palette LOAMA](../images/color-palette.png)
+
+For our font, we've decided to use a monospace font *([Jetbrains Mono](https://www.jetbrains.com/lp/mono/))* for headers, wordmark or highlight text. This allowed us to give of an technological feeling and worked great in creating a contrast with our sans-serif font.
```
Comment (elroyK) [inline project/brand.md]:
```suggestion
For our font, we've decided to use a monospace font *([Jetbrains Mono](https://www.jetbrains.com/lp/mono/))* for headers, wordmark or highlight text. This allowed us to give of a technological feeling and worked great in creating a contrast with our sans-serif font.
```

Comment (Zelzahn) [inline project/brand.md]:
```suggestion
For our font, we've decided to use a monospace font *([Jetbrains Mono](https://www.jetbrains.com/lp/mono/))* for headers, wordmark or highlighted text. This allows us to emit a technological feeling and worked great in creating a contrast with our sans-serif font.
```

File: project/audience.md (hunk 1 above)
Comment (BT-Creator) [reply project/audience.md:15]: Jeremy has been deleted :cry:

**Pull request merged**